src/chrome/content/rules/Radioreference.com.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+	Nonfunctional hosts in *.radioreference.com:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Radioreference.com">
+	<target host="radioreference.com" />
+	<target host="www.radioreference.com" />
+	<target host="api.radioreference.com" />
+	<target host="forums.radioreference.com" />
+	<target host="m.radioreference.com" />
```

Comment:
`proxy.radioreference.com`

Reply:
proxy.radioreference.com is NXDOMAIN according to what I've found. Would you still like me to add it to the ruleset?

Reply:
No, you can ignore this.